src/iOS/Xamarin.iOS/Samples/Utility network/DisplayUtilityAssociations/DisplayUtilityAssociations.cs

```diff
@@ -0,0 +1,228 @@
+                {
+                    return;
+                }
+
+                // Check if the current viewpoint has an extent.
+                Envelope extent = _myMapView.GetCurrentViewpoint(ViewpointType.BoundingGeometry)?.TargetGeometry?.Extent;
+                if (extent == null)
+                {
+                    return;
+                }
+
+                // Get all of the associations in extent of the viewpoint.
+                IEnumerable<UtilityAssociation> associations = await _utilityNetwork.GetAssociationsAsync(extent);
+                foreach (UtilityAssociation association in associations)
+                {
+                    // Check if the graphics overlay already contains the association.
+                    if (_associationsOverlay.Graphics.Any(g => g.Attributes.ContainsKey("GlobalId") && (Guid)g.Attributes["GlobalId"] == association.GlobalId))
+                    {
+                        continue;
+                    }
+
+                    // Add a graphic for the association.
+                    Symbol symbol = null;
+                    if (association.AssociationType == UtilityAssociationType.Attachment) symbol = _attachmentSymbol;
+                    else if (association.AssociationType == UtilityAssociationType.Connectivity) symbol = _connectivitySymbol;
+                    if (symbol != null)
+                    {
+                        Graphic graphic = new Graphic(association.Geometry, symbol);
+                        graphic.Attributes["GlobalId"] = association.GlobalId;
+                        _associationsOverlay.Graphics.Add(graphic);
+                    }
+                }
+            }
+            // This is thrown when there are too many associations in the extent.
+            catch (TooManyAssociationsException)
+            {
+            }
+            catch (Exception ex)
+            {
+                new UIAlertView(ex.GetType().Name, ex.Message, (IUIAlertViewDelegate)null, "OK", null).Show();
+            }
+        }
+
+        public override void LoadView()
+        {
+            // Create the views.
+            View = new UIView() { BackgroundColor = UIColor.White };
+
+            _myMapView = new MapView();
+            _myMapView.TranslatesAutoresizingMaskIntoConstraints = false;
+
+            UIToolbar toolbar = new UIToolbar() { TranslatesAutoresizingMaskIntoConstraints = false };
+
+            _connectivityImage = new UIImageView(new CoreGraphics.CGRect(0, 0, 25, 25));
+            _attachmentImage = new UIImageView(new CoreGraphics.CGRect(0, 0, 25, 25));
+
+            UILabel connectivityLabel = new UILabel() { Text = "Connectivity" };
+            UILabel attachmentLabel = new UILabel() { Text = "Attachment" };
+
+            toolbar.Items = new[]
+            {
+                new UIBarButtonItem() { CustomView = _attachmentImage},
+                new UIBarButtonItem(UIBarButtonSystemItem.FlexibleSpace),
```

Comment:
```suggestion
                new UIBarButtonItem(UIBarButtonSystemItem.FixedSpace) { Width = 8 },
```

I'd suggest a fixed space between the image and the label to make it easier to distinguish at a glance.